charts/README.rst

```diff
@@ -11,15 +11,18 @@ Requires minikube, kubectl and helm.
     $ minikube start
     $ eval $(minikube docker-env)
     $ make -C .. tag
+    $ minikube addons enable coredns
```

Comment:
why?

Reply:
to get the service links across pods working